### PR TITLE
Port TxV2 to wallet

### PR DESCRIPTION
**Pull request by ValentinTrinque**

### CODE REVIEW

File: wallet/handler_deprecated.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	}, nil
+}
+
+func makeNonce() uint64 {
+	max := &big.Int{}
+	// set it to the max value of the uint64
+	max.SetUint64(^uint64(0))
+	nonce, _ := rand.Int(rand.Reader, max)
+	return nonce.Uint64()
+}
```
Comment (jeremyletang) [inline wallet/handler_deprecated.go:67]: Didn't you add a NewNonce method somehwhere?

File: wallet/node_forward.go [hunk 2]
```diff
@@ -140,6 +128,25 @@ func (n *nodeForward) Send(ctx context.Context, tx *SignedBundle, ty api.SubmitT
...
+	req := api.SubmitTransactionV2Request{
+		Tx:   tx,
+		Type: ty,
+	}
+	return backoff.Retry(
+		func() error {
+			clt := n.nextClt()
+			resp, err := clt.SubmitTransactionV2(ctx, &req)
+			if err != nil {
+				return err
```
Comment (jeremyletang) [inline wallet/node_forward.go:141]: Could you use logErrror(err) here?

File: wallet/service.go [hunk 3]
```diff
@@ -137,33 +163,19 @@ func NewServiceWith(log *zap.Logger, cfg *Config, rootPath string, h WalletHandl
 	s.PUT("/api/v1/keys/:keyid/taint", ExtractToken(s.TaintKey))
 	s.PUT("/api/v1/keys/:keyid/metadata", ExtractToken(s.UpdateMeta))
 	s.POST("/api/v1/sign", ExtractToken(s.SignAny))
+	s.POST("/api/v1/command", ExtractToken(s.SignTxV2))
+	s.POST("/api/v1/command/sync", ExtractToken(s.SignTxSyncV2))
```
Comment (jeremyletang) [inline wallet/service.go:167]: Could we default /command to be sync, remove the /command/sync and add a /command/async here? I did the same for the tx v1 recently, as it helps with error returns

**Review by jeremyletang — APPROVED** [review]

File: wallet/handler_deprecated.go (hunk 1 above)
Comment (ValentinTrinque) [reply wallet/handler_deprecated.go:67]: 🎯

**Pull request merged**